trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/VerificationResultTransformer.java

```diff
@@ -187,6 +190,15 @@ public IResult convertTraceAbstractionResult(final IResult result) {
 			final String failurePath = formatTimeSequenceMap(delta2var2value);
 			return new ReqCheckRtInconsistentResult<>(element, plugin, failurePath);
 		}
+		if (spec == Spec.VACUOUS) {
+			// Return a ReqCheckVacuousResult, i.e., a ReqCheckFailResult with additional
+			// VacuityAnnotation containing an invariant that might be enforced by the
+			// vacuous requirement.
+			VacuityAnnotation enforcedInv = VacuityAnnotation
+					.getAnnotation(((BoogieIcfgLocation) element).getBoogieASTNode());
+			return new ReqCheckVacuousResult<>(element, plugin,
+					enforcedInv.getInvariant().toString());
```

Comment:
It would be slightly cheaper to already check here if the invariant is different from `CDD.TRUE` and only then create the string, else use null.